receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/metadata.yaml

```diff
@@ -37,3 +37,11 @@ metrics:
       aggregation: cumulative
       monotonic: false
     attributes: [device, mode, mountpoint, type, state]
+
+  system.filesystem.utilization:
+    enabled: true
+    description: Percentage of filesystem used bytes.
```

Comment:
Trivial but I would use `Percentage of filesystem bytes used.` to be in line with other descriptions